psi_cardinality/c/psi_cardinality_benchmark_c.cpp

```diff
@@ -0,0 +1,191 @@
+#include "absl/strings/str_cat.h"
+#include "benchmark/benchmark.h"
+#include "psi_cardinality_client_c.h"
+#include "psi_cardinality_server_c.h"
+
+namespace psi_cardinality {
+namespace {
+
+void BM_ServerBindingSetup(benchmark::State &state, double fpr) {
```

Comment:
The name can be the same as in the C++ benchmarks, i.e., BM_ServerSetup. The target path will make clear whether we mean C or C++ benchmarks.